### PR TITLE
Add constants AF_RXRPC and PF_RXRPC on Linux

### PR DESCRIPTION
**Pull request by todayman**
See title.  Pretty simple and self-explanatory.

### CODE REVIEW

**Comment by DmitryOlshansky** [comment]
Since these are linux specific no point having them on _posix_ file I guess.
@todayman - could you please follow the convention of putting things into `core.sys.linux.*` if they are linux-specific?

**Comment by todayman** [comment]
There isn't a core.sys.linux.sys.socket.d.  Should I go ahead and create or do you think there's a better place?

**Comment by DmitryOlshansky** [comment]
Time to create ore.sys.linux.sys.socket.d

**Comment by todayman** [comment]
The autotester has been stuck at Pass: 8, Pending: 2 for quite a while now (> 1 week).  What does that mean?  Clicking on the details link shows that there are some failures on Win32 - if those are a real problem I have no way to test / debug those and would appreciate help.

**Comment by yebblies** [comment]
The autotester prioritizes recently-updated pulls, and on some of the slower platforms this means the tester never gets to this pull.  If you rebase on top of master and force-push over this branch it will bump it back up to the top.

**Comment by todayman** [comment]
@DmitryOlshansky core.sys.linux.sys.socket.d is made and the build looks OK.  Is this good to go?

**Comment by schveiguy** [comment]
LGTM

File: src/core/sys/linux/sys/socket.d [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * Authors:   Paul O'Neil
+ */
+module core.sys.linux.sys.socket;
+
+version(linux):
+extern(C):
+@nogc:
+nothrow:
+
+enum {
```
Comment (MartinNowak) [inline src/core/sys/linux/sys/socket.d]: It should publicly `import core.sys.posix.sys.socket;` so it functions as an overlay.

**Comment by todayman** [comment]
@MartinNowak public import added.

**Comment by schveiguy** [comment]
Still looks good, I assume Martin only had that one issue which is now fixed.

**Comment by schveiguy** [comment]
Auto-merge toggled on

**Pull request merged**